openprescribing/frontend/management/commands/measure_definitions/lpzomnibus.json

```diff
@@ -30,6 +30,26 @@
   "numerator_where": [
     "1 = 1 "
   ],
+  "numerator_bnf_codes_query": [
```

Comment:
As this is designed to query the bigquery prescribing dataset (though perhaps it should be querying something smaller like the presentations table?), the column name should be `bnf_code`, not `presentation_code`, I think.

Reply:
It's querying the `bnf` table, so this is not a problem.